src/patchy/api.py

```diff
@@ -216,9 +216,21 @@ def _compile(
         code: str | ast.Module,
         flags: int = 0,
     ) -> CodeType | ast.Module:
+
         result: CodeType | ast.Module = compile(
             code, "<patchy>", "exec", flags=feature_flags | flags, dont_inherit=True
         )
+
+        import linecache
```

Comment:
hoist to module level

Reply:
Totally, this was just a quick and dirty PR to see if you were interested in the idea